gpu-jobs.shtml

```diff
@@ -113,27 +117,30 @@ server, including: </p>
 	</tr>
 	<tr>
 		<td><code>Gpus</code></td>
-		<td>Number of GPUs in an individual job slot on a server(one server can be divided into slots to run multiple jobs)</td>
+		<td>Number of GPUs in an individual job slot on a server (one server can be divided into slots to run multiple jobs).</td>
 	</tr>
 	<tr>
 		<td><code>TotalGPUs</code></td>
-		<td>The total number of GPUs on a server</td>
+		<td>The total number of GPUs on a server.</td>
 	</tr>
 	<tr>
-		<td><code>CUDADriverVersion</code></td>
-		<td>Maximum CUDA runtime version supported by the GPU drivers on the machine. (CUDADriverVersion is not the actual GPU driver version) </td>
+		<td><code>CUDADeviceName</code></td>
+		<td>The type of GPU card.</td>
 	</tr>
 	<tr>
 		<td><code>CUDACapability</code></td>
-		<td>Represents how new the GPU is -- higher numbers are newer GPUs. </td>
+		<td>Represents various capabilities of the GPU. Can be used as a proxy for the GPU card type when 
+		requiring a specific type of GPU. You can see more detail on what the capability numbers mean via 
+		<a href="https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#compute-capabilities">this page
+		on the NVIDIA website</a>.</td>
```

Comment:
I find the CUDA capability values in the table at https://en.wikipedia.org/wiki/List_of_Nvidia_graphics_processing_units to be helpful because it can be looked up for a specific GPU or class of GPUs.  Should we link that as well?